README.md

```diff
@@ -76,11 +76,11 @@ Get the cat model in a service
 import { Injectable } from "@nestjs/common";
 import { InjectModel } from "nestjs-typegoose";
 import { Cat } from "./cat.model";
-import { ModelType } from "@typegoose/typegoose";
+import { ReturnModelType } from "@typegoose/typegoose";
 
 @Injectable()
 export class CatsService {
-  constructor(@InjectModel(Cat) private readonly catModel: ModelType<Cat>) {}
+  constructor(@InjectModel(Cat) private readonly catModel: ReturnModelType<Cat>) {}
```

Comment:
Does `ReturnModelType` works properly? For me it was giving some error. May be i am using old `nestjs-typegoose` package? I am not sure! Please test this. Thanks.

Reply:
Right, there was one issue - has to be ```typeof Cat```